test/integration/targets/setup_passlib_controller/runme.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# Requirements have to be installed prior to running ansible-playbook
+# because plugins and requirements are loaded before the task runs
+python -m pip install passlib
```

Comment:
This isn't using a virtual environment, which is going to be problematic in the not-too-distant future. It's probably time for me to add support for ansible-test managed virtual environments for integration tests, which would greatly simplify this and avoid the issue of working without a virtual environment.